### PR TITLE
fix: add documentation for earth engine import

### PR DESCRIPTION
**Pull request by jenniferarnesen**

### CODE REVIEW

File: src/user/importexport-app.md [hunk 1]
```diff
@@ -133,49 +132,104 @@ Import.
...
 
-### GML Import { #gml_import } 
+### Earth Engine Import { #ee_import }
 
-GML Import can be accessed from the sidebar by clicking on GML
+Earth Engine Import can be access from the sidebar by clicking on Earth Engine
 Import.
 
-![](resources/images/import_export/gml_import.png)
+Google Earth Engine (WorldPop) data is fetched using the Google Earth Engine API. A Google
```
Comment (turban) [inline src/user/importexport-app.md]: Suggested change: "Import high resolution population data from WorldPop using Google Earth Engine.

File: src/user/importexport-app.md [hunk 2]
```diff
@@ -133,49 +132,104 @@ Import.
...
+EarthEngine account is required to use this importer.
```
Comment (turban) [inline src/user/importexport-app.md]: Add space between "Earth Engine" + add a link to https://docs.dhis2.org/en/topics/tutorials/google-earth-engine-sign-up.html

File: src/user/importexport-app.md [hunk 3]
```diff
@@ -133,49 +132,104 @@ Import.
...
+
+The first section of the form is used to configure the Earth Engine data to import.
+
+1. Select which Earth Engine dataset should be imported. Currently the choices are _Population_ and _Population age groups_.
+
+2. After a dataset has been selected, a period must be selected. Only one period can be imported at a time.
+
+3. Choose how the data should be rounded. By default data is not rounded.
+
+4. Select which organisation units to import data for. If you select facility level organisation units, then you must choose an associated geometry for the facilities. Without an associated geometry for facilities, the Earth Engine cannot determine the population.
```
Comment (turban) [inline src/user/importexport-app.md]: "to import data for" sounds Norwegian English but I trust native speakers :-)

File: src/user/importexport-app.md [hunk 4]
```diff
@@ -133,49 +132,104 @@ Import.
...
+
+![](resources/images/import_export/ee_ou_associated_geometry.png)
+
+#### Select the data elements to import the Earth Engine data into
+
+Once you have configured the Earth Engine dataset, you need to select which data element to import the data into. For datasets with disaggregation groups, such as "Population age groups", the DHIS2 data element must have disaggregations in the form of category option combos that match the Earth Engine dataset disaggregation groups.
```
Comment (turban) [inline src/user/importexport-app.md]: "to import the data into" also sounds a bit Norwegian English to me :-)

Comment (jenniferarnesen) [reply src/user/importexport-app.md]: I think I've lived here too long 😛 I don't know my prepositions anymore. Also, from a pedantic standpoint sentences should not end with a preposition, but I hear that is outdated thinking 🤷

File: src/user/importexport-app.md [hunk 5]
```diff
@@ -133,49 +132,93 @@ Import.
...
+    - Org unit ID scheme
+    - ID scheme
 
 3.  Click on the **Import** button which will upload the file and start the
     importing process.
 
-### GML Import { #gml_import } 
+### Earth Engine Import { #ee_import }
+
+Earth Engine Import can be access from the sidebar by clicking on Earth Engine
```
Comment (janhenrikoverland) [inline src/user/importexport-app.md]: can be accessed

File: src/user/importexport-app.md (hunk 4 above)
Comment (janhenrikoverland) [reply src/user/importexport-app.md]: I think it sounds okay.

File: src/user/importexport-app.md [hunk 6]
```diff
@@ -133,49 +132,93 @@ Import.
...
+
+![](resources/images/import_export/ee_group_coc_mapping.png)
+
+> **Configuring data elements for Earth Engine import**
+>
+> When an Earth Engine dataset contains disaggregation groups, the DHIS2 data element must be configured with corresponding category option combos. For example the "Population age groups" dataset is disaggregated by gender (Male, Female) and 5-year age groups.
+>
+> In DHIS2 this means that you must have a Male/Female category, and a 5-year age group category (<1yr, 1-4yr, 5-9yr, 10-14yr... 80+yr). These are combined into a category combination.
+>
+> Pro-tip: To automatically match the category option combo to the earth engine disaggregation group, add a Code to each category option combo that matches the earth engine group name. For example, with "Population age groups", the groups are named: F_0, F_1, F_5..., M_0, M_1, M_5...
```
Comment (janhenrikoverland) [inline src/user/importexport-app.md]: Earth Engine, not earth engine.

File: src/user/importexport-app.md [hunk 7]
```diff
@@ -133,49 +132,93 @@ Import.
...
+
+#### Run the import
+
+Once the data element and category option combos have been selected, The Preview button is enabled. After reviewing the data you want to import, you can do a dry run first, or proceed with the actual import.
```
Comment (janhenrikoverland) [inline src/user/importexport-app.md]: the Preview

Comment (jenniferarnesen) [reply src/user/importexport-app.md]: You mean:
The Preview button => The Preview
?

Comment (janhenrikoverland) [reply src/user/importexport-app.md]: No, just lowercase t in "the"

**Review by janhenrikoverland — APPROVED** [review]

**Pull request merged**